src/net/libwsIO.cpp

```diff
@@ -200,7 +202,7 @@ void LibwsClient::websockMsgCb(ws_t ws, char *msg, uint64_t len, int binary, voi
                          
 bool LibwsClient::wsSendMessage(char *msg, size_t len)
 {
-    assert (mWebSocket);
+//    assert (mWebSocket);
```

Comment:
Please, don't comment/remove asserts without a good reason

Reply:
This one was occurring for me fairly regularly at the time, though that may have been before I got the third party dependencies right.  I've done a bit of testing and it's not occurring now.  I've also reviewed the differences vs current `develop` branch, and this was the only one.  I'll restore this line, sorry about that.  cheers